src/app/components/navigation/IsaacApp.tsx

```diff
@@ -142,6 +144,7 @@ export const IsaacApp = () => {
 
     // Render
     return <Router history={history}>
+        <ErrorClearingRouter>
```

Comment:
Due to the lack of indentation of the children, it's not clear that this is a wrapper. Again, this looks like an ESLint issue.

Reply:
I spotted this one, but if I run this file through prettier it changes most lines of the code due to existing indentation issues - happy to do that, just less clear in PR what was changed

Reply:
You've done the right thing. I was only highlighting this, but it is something that should be tackled in its own ticket. If you run prettier and it changes a lot of files, I think that means that the prettier rules are not integrated with ESLint. So I'd say we need a ticket to fix that. Once that's done, these errors will be detected as part of the pipeline.